build fails

I installed npm packages and did the build, but it fails with many error on my windows computer:

```
....
  74:80  error  Delete `␍`  prettier/prettier
  75:17  error  Delete `␍`  prettier/prettier
  76:25  error  Delete `␍`  prettier/prettier
  77:20  error  Delete `␍`  prettier/prettier
  78:16  error  Delete `␍`  prettier/prettier
  79:17  error  Delete `␍`  prettier/prettier
  80:3   error  Delete `␍`  prettier/prettier

✖ 80 problems (80 errors, 0 warnings)
  80 errors and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/config/proxy.js 26:0-65 30:28-39 31:23-34
 @ ./src/config/index.js
 @ ./node_modules/babel-loader/lib!./node_modules/eslint-loader/dist/cjs.js!./src/app.js
 @ ./src/app.js-exposed

ERROR in ./src/config/index.js
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

D:\glance\paraview-glance\src\config\index.js
  1:54  error  Delete `␍`  prettier/prettier
  2:1   error  Delete `␍`  prettier/prettier
  3:17  error  Delete `␍`  prettier/prettier
  4:9   error  Delete `␍`  prettier/prettier
  5:3   error  Delete `␍`  prettier/prettier

✖ 5 problems (5 errors, 0 warnings)
  5 errors and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/app.js (./node_modules/babel-loader/lib!./node_modules/eslint-loader/dist/cjs.js!./src/app.js) 18:0-48 49:63-69
 @ ./src/app.js-exposed

ERROR in ./src/config/proxy.js
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):
...
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,1 @@
+* text=auto eol=lf

--- a/prettier.config.js
+++ b/prettier.config.js
@@ -3,4 +3,5 @@ module.exports = {
   singleQuote: true,
   trailingComma: 'es5',
   arrowParens: 'always',
+  endOfLine: 'lf',
 };

